### ISSUE TITLE
Graviton mediation request from Metagame 

### ISSUE THREAD

**Issue by tamarandom** [TECommons/coordination#445]
## 📋 Anything to add?

How can we help out here? 

kyle stargarden.ai 🐙: Any folks from Graviton want to volunteer to mediate a dispute that has happened in Metagame?

kyle stargarden.ai 🐙: Perhaps @liviade could make a recommendation?  I am interested to reach out for help on this issue.  It's been a significant drain on community morale and energy.

## 🎉 Subtasks

- [ ] 
- [ ] 
- [ ] 

## 🤼‍ Reviewer

@

**Comment by JuankBell** [comment]
I'm reaching out!

**Comment by JuankBell** [comment]
As there was no previous criteria to frame the situation within the org, involved parts had a subjective point of view that made it difficult to build a bridge or reach an agreement, that is why the approach taken was to listen and document future actions for each part to prevent this from happening again. 

Relations between Gravity and Metagame members were built, and the suggestion we have made is to contribute in the design of Covenants, codes of conduct & other documents that could serve as a criteria to frame the expected rules and boundaries in the organization and be applied with clear ways of answering to the behavior of agents.

I close this issue, but find this as a great starting point for further collaboration between Metagame and gravity-TEC, to promote Alternative Dispute Resolution and also to help on designing documents that can be applied as a criteria to regulate behavior in DAOs

Hopefully we can present Gravity at the Metafest. I suggested this schedule **Wed 24th, at 11amEST - 17:00 PM CET**. but i haven't received confirmation 

![Captura de Pantalla 2021-03-18 a la(s) 7.02.21 a. m..png](https://images.zenhubusercontent.com/5fb57c9c5472cf79e80e7f7e/107b1f30-b6d3-4bb4-b47f-228d55f78414)
